tests/parsers/roche_cedex_hires/testdata/roche_cedex_hires_example_3.json

```diff
@@ -2386,17 +2386,41 @@
                                             "value": 0.01331124444,
                                             "unit": "10^6 cells/mL"
                                         },
+                                        "average live cell diameter (cell counter)": {
```

Comment:
Note to reviewer: this was a bug in original implementation, turns out walrus operator evaluates 0 as falsey!!!